### PR TITLE
Featdet help fix

### PR DESCRIPTION
**Pull request by omokshyna**
Removed offline help from several feature detection, filtering and merging modules. Minor changes in code for merging dialog GUI improvement.
**Please**, merge after this documentation PR is merged - https://github.com/mzmine/mzmine_documentation/pull/53

### CODE REVIEW

**Comment by omokshyna** [comment]
Corresponding documentation PR is merged.

File: src/main/java/io/github/mzmine/modules/dataprocessing/featdet_mobilityscanmerger/MobilityScanMergerParameters.java [hunk 1]
```diff
@@ -41,10 +41,10 @@ public class MobilityScanMergerParameters extends SimpleParameterSet {
       MZmineCore.getConfiguration().getIntensityFormat(), 1E1, 0d, 1E12);
 
   public static final ComboParameter<MergingType> mergingType = new ComboParameter<>("Merging type",
-      "merging type", MergingType.values(), MergingType.SUMMED);
+      "Spectra merging algorithm", MergingType.values(), MergingType.SUMMED);
 
   public static final ComboParameter<Weighting> weightingType = new ComboParameter<>(
-      "m/z weighting", "Weights m/z values by their intensities with the given function.",
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/modules/dataprocessing/featdet_mobilityscanmerger/MobilityScanMergerParameters.java:47]: should be mz weight or mz weighting. (Weighing is to actually put it on a scale right to determine the weight?)

https://en.wikipedia.org/wiki/Weighting

**Review by robinschmid — COMMENTED** [review]
looks good only one small comment

File: src/main/java/io/github/mzmine/modules/dataprocessing/featdet_mobilityscanmerger/MobilityScanMergerParameters.java (hunk 1 above)
Comment (omokshyna) [reply src/main/java/io/github/mzmine/modules/dataprocessing/featdet_mobilityscanmerger/MobilityScanMergerParameters.java:47]: You're right, fixed.

**Pull request merged**